src/client.c

```diff
@@ -0,0 +1,192 @@
+#include "client.h"
+#include "blokus.h"
+#include <ncurses.h>
+#include <stdlib.h>
+#include <string.h>
+
+int choose_tile_handler(int c, rcb_t *rcb, coord_t *coord)
+{
+    if (rcb->render_player == 0) 
+        recover_tile_preview(rcb->gcb, tile_relation[coord->y][coord->x]);
+    switch (c) {
+    case 'h':
+    case 'H':
+    case KEY_LEFT:
+        if (coord->x > 0) {
+            coord->x--;
+            if (coord->x > 0 
+                && tile_relation[coord->y][coord->x] 
+                == tile_relation[coord->y][coord->x+1]) coord->x--;
```

Comment:
We need spaces around binary operators QAQ
```suggestion
                == tile_relation[coord->y][coord->x + 1]) coord->x--;
```